app.js

```diff
@@ -1,8 +1,85 @@
 // import functions and grab DOM elements
+import { renderGoblin } from './render-utils.js';
+
+const defeatCounterEl = document.querySelector('#defeat-counter');
+const playerHPEl = document.querySelector('#player-hp');
+const playerImgEl = document.querySelector('#player-img');
+const form = document.querySelector('form');
+const goblinsListEl = document.querySelector('.goblins');
 
 // let state
+let defeatedGoblins = 0;
+let playerHP = 11;
+let goblins = [
+    { name: 'Loki', hp: 3, strength: 1 },
+    { name: 'Young Thanos', hp: 9, strength: 5 },
+];
 
 // set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
+form.addEventListener('submit', (event) => {
+    event.preventDefault();
+
+    const data = new FormData(form);
+    //const goblinName = data.get('goblin-name');
+    const newGoblin = {
+        name: data.get('goblin-name'),
+        hp: Math.ceil(Math.random() * 10),
+        strength: Math.ceil(Math.random() * 5),
+    };
+    goblins.unshift(newGoblin);
+
+    displayGoblins();
+
```

Comment:
if you add `form.reset()` here, it will clear out the input field once the user submits the new goblin's name